lib/Target/AMDGPU/SILowerControlFlow.cpp

```diff
@@ -473,6 +481,53 @@ void SILowerControlFlow::combineMasks(MachineInstr &MI) {
     MRI->getUniqueVRegDef(Reg)->eraseFromParent();
 }
 
+bool SILowerControlFlow::shouldSkip(const MachineBasicBlock &From,
```

Comment:
Used the same name for these functions 'shouldSkip' and 'SkipMaskBranch' here as well. Let me know if you want me to have different names.